tools/benchmarking/benchmark.py

```diff
@@ -225,6 +225,8 @@ def distribute(config: Union[DictConfig, ListConfig]):
         distribute_over_gpus(config, folder=runs_folder)
     if "wandb" in config.writer:
         upload_to_wandb(team="anomalib", folder=runs_folder)
+    if "comet" in config.writer:
+        upload_to_comet(folder=runs_folder)
```

Comment:
I see that comet uses `workspaces` as entities to group runs. Can you add that to the parameters so that we can create a workspace for the `anomalib` team and store all the runs there?

Reply:
hey @ashwinvaidya17. 

Comet workspaces are meant to be unique to the platform.  If i try

COMET ERROR: Workspace anomalib doesn't exist.
when i go try and create the anomalib workspace i get "error workspace name already exists".

Who ever has the anomalib workspace would need to add users to the workspace for me to have my projects shown there. This would be big hoop to jump if I were an open-source user of anomalib just wanting to use the BM tool. Not sure what value it is adding

Checkout my BM project that I shared!

Reply:
I created a workspace named anomalib but I think it is better if skip workspace for now. I think this part will require refactoring the benchmarking script from my side.